sections/dir.zsh

```diff
@@ -21,14 +22,25 @@ SPACESHIP_DIR_COLOR="${SPACESHIP_DIR_COLOR="cyan"}"
 spaceship_dir() {
   [[ $SPACESHIP_DIR_SHOW == false ]] && return
 
-  local dir
+  local dir trunc_prefix
 
   # Threat repo root as a top-level directory or not
   if [[ $SPACESHIP_DIR_TRUNC_REPO == true ]] && spaceship::is_git; then
     local git_root=$(git rev-parse --show-toplevel)
-    dir="$git_root:t${$(expr $(pwd) : "$git_root\(.*\)")}"
+
+    if [[ $git_root:h == / ]]; then
+      trunc_prefix=/
```

Comment:
If I understand correctly, This checks whether the `git_root` is `/`, Right ? But why would someone keep `/` under `git` !! 😕

Reply:
This checks whether the *parent* of `git_root` is `/`. For strange people who keep their repos directly under the root (I'm not doing that personally).

Reply:
Sorry I missed that `:h` modifier. Not judging anyone, Still it feels kinda weird to store repositories directly under root.